pkg/arvo/app/metadata-store.hoon

```diff
@@ -73,15 +89,14 @@
     |=  =path
     ^-  (unit (unit cage))
     ?+  path  (on-peek:def path)
-        [%x %all ~]               ``noun+!>(associated)
-        [%x %group-indices ~]     ``noun+!>(group-indices)
-        [%x %app-name-indices ~]  ``noun+!>(app-name-indices)
-        [%x %app-path-indices ~]  ``noun+!>(app-path-indices)
+        [%y %associations ~]      ``noun+!>(associations)
+        [%y %group-indices ~]     ``noun+!>(group-indices)
+        [%y %app-indices ~]       ``noun+!>(app-indices)
+        [%y %resource-indices ~]  ``noun+!>(resource-indices)
```

Comment:
I, uh, don't think this is how you use `%y` cares? They're for getting a "directory listing" of the given path. In context of apps, I think this means that if you support a `/something/[your-resource]` scry, then doing a `%y` scry on `/something` will give you all values of `your-resource` for which there's something to be scried out using the more specific path.

ie you want to give _keys_, not _values_. In that regard, certainly the `/associations` one here should be a `%x`, and probably the other ones as well. If you want to actually implement a `%y`, you'd do `~(key by associations)` etc.

Reply:
Done, thank you.

Reply:
If it's going to return full keys rather than path segments, maybe it should be %t rather than %y.